torch/nn/modules/_functions.py

```diff
@@ -96,3 +96,47 @@ def backward(self, grad_output):
             grad_bias = None
 
         return grad_input, grad_weight, grad_bias, None, None, None, None, None, None
+
+    @staticmethod
```

Comment:
This should be a class method. Line 128 should use `cls` instead

Reply:
Will Fix that.